quarto-render/action.yaml

```diff
@@ -0,0 +1,13 @@
+name: "Quarto Render"
+branding:
+  icon: 'award'  
+  color: 'blue'
+description: "Renders a repository of documents with Quarto"
+runs:
+  using: "composite"
+  steps: 
+    - name: "Install Quarto"
+      uses: quarto-dev/quarto-actions/install-quarto@v1
+    - run: | 
+        quarto render
+      shell: bash
```

Comment:
Do you think we should allow pass some arguments ? 

Maybe it is not that useful and not as easy to do...

Reply:
We can definitely do that! I was actually working on being able to pass the output format to the render function in the action - [here is my work in progress](https://github.com/pommevilla/quarto-render/blob/feature/render-to/action.yaml). 

The use case was that I wanted the action to be able to do `quarto render --to all` so that the pdf, website, and docx would be render on push to the repo. I ran into some trouble with tinytex at run time, though (here is a kind of related issue on the [Workbench site](https://github.com/rstudio/rstudio-pro/issues/2958)). I'll spend some time on it this week and see what all needs to be done.

Another input argument that I'm working on is rendering in a specific subdirectory, but that's less a `render` argument and more a Github action argument. Did you have other arguments in particular you think would be helpful?

Reply:
I was thinking about passing any argument really but afterall in that case, one could use directly 
````
- name: "Install Quarto and render"
  uses: quarto-dev/quarto-actions/render-quarto@v1
- name: rendering quarto
  run: |
    quarto render <arg>
````

So maybe the action quarto-render should stay focus on some specific argument only. We can see later which arguments are useful. I was thinking about parameters, cache, pandoc args if any. 
But you can also pass those in YAML so seems ok. 

`--to all` can indeed be usefule